server/athenian/api/typing_utils.py

```diff
@@ -89,9 +89,7 @@ def is_union(klass: object) -> bool:
 
 def is_optional(klass: type):
     """Determine whether klass is an Optional."""
-    return (
-        is_union(klass) and len(klass.__args__) == 2 and issubclass(klass.__args__[1], type(None))
-    )
+    return is_union(klass) and types.NoneType in klass.__args__
```

Comment:
The fresh native code I added in `web_model_io.pyx` has a hard requirement for `None` to come second and uses this function. We should check both the expected args length (2) and that None comes second.

Reply:
Reverted,
This is needed for `Optional[int | float]` (or `Optional[Union[int, float]]`) to work, I solved this with a workaround